src/components/header/Header.tsx

```diff
@@ -14,8 +15,12 @@ export const Header = () => {
     <Styled.Container isActive={isActive}>
       <Styled.Logo>team awesome</Styled.Logo>
       <Styled.Menu isActive={!isActive}>
-        <Styled.MenuItems>Timesheets</Styled.MenuItems>
-        <Styled.MenuItems>Team members</Styled.MenuItems>
+        <Link href="/time-sheets" passHref>
```

Comment:
It only makes sense to add the `passHref` property if the child of your `<Link />` tag is an `<a />` tag: https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag.